### PR TITLE
update privacy policy

### PR DESCRIPTION
**Pull request by CaraDune**
Replacing old privacy policy provided by AgID with new policy provided by PagoPA as data controller for processing of developers' personal data. 

Closed the stylesheet tag that prevented from loading applicable css rules. Introduced additional style rules to allow indented unordered list. 
Attaching screenshots providing preview of code for browsers, tablets and mobile.

This is my first PR so feedback is welcome. 

Preview_Browser

Preview_Tablet

Preview_Mobile

### CODE REVIEW

File: web/privacy.html [hunk 1]
```diff
@@ -1,19 +1,22 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="it">
 
 <head>
   <meta charset="utf-8">
-  <meta http-equiv="x-ua-compatible" content="ie=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```
Comment (balanza) [inline web/privacy.html:6]: Why this change?

File: web/privacy.html [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
+  <link href="//fonts.googleapis.com/css?family=Titillium+Web:400,400italic,700," rel="stylesheet" type="text/css"/>
+  <link href="//italia.github.io/design-web-toolkit/build/build.css" rel="stylesheet" media="all" type="text/css"/>
+  <link href="//pagopa.github.io/io-developer-portal-backend/global.css" rel="stylesheet" media="all" type="text/css"/>
+  <style>
+      ul, ol {
+          list-style: initial;
+          margin-left: 20px;
+      }
+      </style>
   <!-- <link href="./global.css" rel="stylesheet" media="all" type="text/css" /> -->
```
Comment (balanza) [inline web/privacy.html:17]: We can remove commented code

File: web/privacy.html [hunk 3]
```diff
@@ -22,64 +25,157 @@
...
+        </p>
+        <br>
+        <p>
+        <h3>Data protection officer (DPO) o Responsabile Protezione Dati (RPD)</h3>
+        La società PagoPA Spa ha nominato un proprio Responsabile della Protezione dei dati, ai sensi dell’art. 37 del Regolamento UE 2016/679, che può essere contattato ai seguenti recapiti:
+        <ul>
+            <li>mail: <a href="mailto:dpo@pagopa.it">dpo@pagopa.it</a>;</li>
+        <li>PEC: <a href="mailto:dpo@pec.pagopa.it">dpo@pec.pagopa.it</a>;</li>
+        <li>indirizzo: Via Sardegna n. 38 - 00187, ROMA (sede operativa della società)</li>
+</ul>
```
Comment (balanza) [inline web/privacy.html:41]: Please check document indentation

Comment (CaraDune) [reply web/privacy.html:41]: Tag position (which is messy on this case) does not affect formatting, so it is more a beauty change. I will tide up the code anyway to make it more readable for review and future changes. Thanks!

File: web/privacy.html (hunk 1 above)
Comment (CaraDune) [reply web/privacy.html:6]: The last attribute ("shrink-to-fit=no") serves the purpose to improve viewport's rendering from mobile. In a nuthsell: viewport meta tags having "width=device-width" cause the page to scale down to fit content that overflows the viewport bounds. You can override this behavior by adding "shrink-to-fit=no", preventing the page from scaling to fit the viewport.
Such added value is compliant with Bootstrap Italia's library requirements(see [here](https://italia.github.io/bootstrap-italia/docs/come-iniziare/introduzione/#utilizzo-come-dipendenza) under Meta tag responsive)

**Comment by francescopersico** [comment]
@balanza @CaraDune 
Take in consideration that the site is served from the gh-pages branch that is currently not aligned with master.

**Comment by CaraDune** [comment]
thank you @francescopersico , I added you as a reviewer to collect your input on the task regarding alignment of gh-pages branch with master

**Comment by balanza** [comment]
Do we have any blocker for this?

**Review by gunzip — APPROVED** [review]

**Comment by CaraDune** [comment]
@balanza 

> Do we have any blocker for this?

yes and no:
_not blocking (substantially)_ because the task is to update the content of the policy, which is something we will achieve with the merge.
_blocking (aesthetically)_ because if we don't align gh-pages branch with master as @francescopersico noted, css style rules will not be visible (I attributed this to an open style tag, but it was not the case) and policy will be displayed in a messy/ugly fashion.

is there someone who can proceed to such alignment after merge or guide me in doing so? cc @gunzip

**Comment by francescopersico** [comment]
@CaraDune @balanza we can merge this and apply the changes also to the gh-pages branch.

**Comment by balanza** [comment]
Sorry I don't understand. Are those files published from the `gh-pages` branch? If so, isn't more correct to merge this PR against such branch instead of against master?

**Comment by francescopersico** [comment]
> Sorry I don't understand. Are those files published from the `gh-pages` branch? If so, isn't more correct to merge this PR against such branch instead of against master?

IMO it's always better/easier to merge a PR to the branch it was detached from. Branch created from master return to master.
The main problem here is that the gh-pages branch contains a not updated version of the files. For example, this PR https://github.com/pagopa/io-developer-portal-backend/pull/83 is not included, instead, it's included in master.
Use the way you want just be sure that at the end both master and gp-pages branches contain the updated version of the files.

**Comment by cloudify** [comment]
Congrats on your first PR @CaraDune , it's a great achievement!

**Comment by balanza** [comment]
So I'd just merge this and make another PR against gh-pages

**Review by francescopersico — APPROVED** [review]

File: web/privacy.html (hunk 1 above)
Comment (CaraDune) [reply web/privacy.html:6]: @balanza yes I removed intentionally. The tag itself serves the purpose to improve rendering for older versions of IE. I removed it because I did not find any reference to this attribute in Bootstrap Italia's library. It is used in io.italia.it homepage because (I suppose) it is possible that the public at large may use such older versions of IE. I am reinserting it in the new PR though
